packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -906,6 +908,26 @@ def _check_class_attributes(self) -> None:
         if not hasattr(self, "synchronized_data_class"):
             logging.warning(f"No `synchronized_data_class` set on {self}")
 
+    @classmethod
+    def auto_round_id(cls) -> str:
+        """
+        Get round id automatically.
+
+        This method returns the auto generated id from the class name if the
+        class variable behaviour_id is not set on the child class.
+        Otherwise, it returns the class variable behaviour_id.
+        """
+        return (
+            cls.round_id
+            if isinstance(cls.round_id, str)
+            else cls.__pattern.sub("_", cls.__name__).lower()
+        )
```

Comment:
Same concern about conflicts as on the behaviour side. Does it make sense to include the module name?

Reply:
Same response